pkg/liquidity-mining/contracts/gauges/arbitrum/ArbitrumRootGauge.sol

```diff
@@ -45,7 +45,7 @@ contract ArbitrumRootGauge is StakelessGauge {
 
     function _postMintAction(uint256 mintAmount) internal override {
         // Token needs to be approved on the gateway NOT the gateway router
-        _balToken.approve(_gateway, mintAmount);
+        _balToken.approve(_gateway, mintAmount); // ERC20 compliant; no need to use safeApproval.
```

Comment:
I'd just use `safeApprove` everywhere

Reply:
It's easier to check and we can skip these comments

Reply:
Done.